packages/cli/lib/projects.js

```diff
@@ -232,11 +232,102 @@ const pollBuildStatus = async (accountId, name, buildId) => {
   });
 };
 
+const pollDeployStatus = async (accountId, name, deployId) => {
```

Comment:
This is very similar to the build status polling logic. Will definitely DRY this up- time permitting, I'll do that in this PR